src/app/address/fields.js

```diff
@@ -22,17 +29,31 @@ module.exports = {
   "address-search": {
     type: "text",
     autocomplete: "Postcode",
+    formatter: [
+      {
+        type: "removeSpaces",
+        fn: (val) => val.replace(/\s+/g, ""),
+      },
+    ],
     validate: [
```

Comment:
Added validation into the FE rather than just the backend because it made sense to use the existing framework we're already using.